packages/block-editor/src/components/block-mover/index.native.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { first, last, partial, castArray } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { ToolbarButton } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { withInstanceId, compose } from '@wordpress/compose';
+
+const BlockMover = ( {
+	isFirst,
+	isLast,
+	onMoveDown,
+	onMoveUp,
+} ) => (
+	<>
+		<ToolbarButton
+			accessibilityLabel={ __( 'Move up' ) }
```

Comment:
Is the `accessibilityLabel` needed if it's the same as `label`? If it is, then it's missing from the `Move down` button

Reply:
+1 about the accessibility labels concern.
Current develop has labels with more information, that (I think) are also used on UI Testing.
https://github.com/wordpress-mobile/gutenberg-mobile/blob/40c1374072a37b5bdf4c8730af3b68f5a5fb0f09/src/block-management/inline-toolbar/index.js#L58-L74

And the prop to set the accessibility label is actually `title`.

Reply:
Thanks for noticing, updating this 👍